website/source/docs/platform/k8s/helm.html.md

```diff
@@ -443,84 +443,50 @@ consul-server-2                            10.60.2.197:8301  alive   server  1.4
 
 ## Helm Chart Examples
 
-The below values.yaml can be used to set up a single server Consul cluster with a LoadBalancer to allow external access to the UI and API.
+The below `values.yaml` can be used to set up a single server Consul cluster with a `LoadBalancer` to allow external access to the UI and API.
 
-```
+```yaml
 global:
   enabled: true
-  image: "consul:1.4.2"
```

Comment:
I've deleted a lot of the keys here that are on by default. This simplifies the example.

Reply:
The default is set to the latest version though: https://github.com/hashicorp/consul-helm/blob/master/values.yaml#L19

Reply:
```
$ cat test.yaml
global:
  enabled: true

server:
  replicas: 1
  bootstrapExpect: 1

ui:
  service:
    type: LoadBalancer

$ helm template -x templates/server-statefulset.yaml -f test.yaml . | grep image
          image: "consul:1.5.0"
```